.laminas-ci.json

```diff
@@ -4,7 +4,7 @@
         "filter"
     ],
     "ignore_php_platform_requirements": {
-        "8.4": false
+        "8.5": true
```

Comment:
What prevents 8.5 installation? Is it 3rd party package or only ours?

Reply:
All laminas deps in require. Probably some or all dev deps